src/components/Modals/PrismicDocumentModal/PrismicDocumentModal.js

```diff
@@ -36,8 +36,7 @@ import type { ScrollEvent, LayoutEvent } from 'utils/types/react-native';
 import { spacing } from 'utils/variables';
 
 // Services
-import * as Prismic from 'services/prismic';
-
+import { getPrismicDocumentAsHTML } from 'services/cms/PrismicPrivacyTermsContent';
```

Comment:
See my comment below - it seems that we're using a function from another service named Privacy Terms Content... is it worth making this a generic function instead of pulling the method `getPrismicDocumentAsHTML ` out of `PrismicPrivacyTermsContent` each time? The term Privacy Policy confused me...

Reply:
Done.